modules/common-attributes.adoc

```diff
@@ -23,7 +23,10 @@ endif::[]
 :kebab: image:kebab.png[title="Options menu"]
 :rh-openstack-first: Red Hat OpenStack Platform (RHOSP)
 :rh-openstack: RHOSP
-:console-redhat-com: Red Hat OpenShift Cluster Manager
+:cluster-manager-first: Red Hat OpenShift Cluster Manager
+:cluster-manager: OpenShift Cluster Manager
+:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager]
+:cluster-manager-url-pull: link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site]
```

Comment:
For the text that is visible in the doc, does it currently read "pull secret from the Red Hat OpenShift Cluster Manager site"?

If yes, this should be changed to "pull secret from Red Hat OpenShift Cluster Manager". We don't refer to it as a 'site' anywhere.